app/src/main/java/com/example/cooktaildb/ui/detail/DetailDrinkActivity.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.cooktaildb.ui.detail
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import com.example.cooktaildb.base.BaseActivity
+import com.example.cooktaildb.databinding.ActivityDetailDrinkBinding
+
+class DetailDrinkActivity : BaseActivity<ActivityDetailDrinkBinding>(ActivityDetailDrinkBinding::inflate) {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+    }
+
+    companion object {
+        fun getIntent(startFragment: Context) = Intent(startFragment, DetailDrinkActivity::class.java)
```

Comment:
```suggestion
        fun getIntent(startActivity: Context) = Intent(startFragment, DetailDrinkActivity::class.java)
```
Cơ mà sao lại define thêm hàm này ấy @hungnd-3027 nhỉ?
Mình chỉ nên define khi pass thêm data kèm theo ``Intent``

Reply:
@daolq-2712 Giờ e sửa lại đoạn này ạ

Reply:
@hungnd-3027 
Cơ mà anh thấy giữ lại cũng hay, mình dễ inspect được Class này được gọi từ những nơi nào!

Em cứ keep lại nhé!